libpod/container_internal_linux.go

```diff
@@ -635,7 +635,7 @@ func (c *Container) setupSystemd(mounts []spec.Mount, g generate.Generator) erro
 			Destination: "/sys/fs/cgroup/systemd",
 			Type:        "bind",
 			Source:      "/sys/fs/cgroup/systemd",
-			Options:     []string{"bind", "nodev", "nosuid", "rprivate"},
+			Options:     []string{"bind", "nodev", "noexec", "nosuid", "rprivate"},
```

Comment:
Any chance we can grab flags from the host mount to make sure they match?

Reply:
I was just starting to look to see if this was possible! This would be the preferred solution.

Reply:
That would be nice, but I would prefer to do that in a different PR and get this one merged.

Reply:
Do we want to track this work in an issue or something to make sure it doesn't drop through the cracks?

Reply:
@dustymabe I made a quick and dirty issue: https://github.com/containers/podman/issues/7661